.gitignore

```diff
@@ -255,3 +255,4 @@ _templates/
 
 # Test Results
 *.trx
+/Ocelot.sln.DotSettings

```

Comment:
Note 1

Reply:
@ggnaegi 
Could you send me a full list of ReSharper artifacts being added as standard settings by the tool to .NET solutions please?

I'll add them to ignore list...
In future, if someone will convert, or create std setup of ReSharper tools then we readd them and remove from **.gitignore** file.
But for now, I want to make sure that contributors don't change line-endings at all. This is critical settings for code review.

Reply:
What's your vision to fix the problem with line-endings in ReSharper?